drivers/windows/dir_access_windows.cpp

```diff
@@ -340,6 +361,33 @@ String DirAccessWindows::get_filesystem_type() const {
 	ERR_FAIL_V("");
 }
 
+bool DirAccessWindows::is_case_sensitive(const String &p_path) const {
+	String f = p_path;
+	if (!f.is_absolute_path()) {
+		f = get_current_dir().path_join(f);
+	}
+	f = fix_path(f);
+
+	HANDLE h_file = ::CreateFileW((LPCWSTR)(f.utf16().get_data()), 0,
+			FILE_SHARE_READ | FILE_SHARE_WRITE | FILE_SHARE_DELETE,
+			nullptr, OPEN_EXISTING, FILE_ATTRIBUTE_NORMAL, nullptr);
```

Comment:
```suggestion
			nullptr, OPEN_EXISTING, FILE_FLAG_BACKUP_SEMANTICS, nullptr);
```
I was getting ACCESS_DENIED error here, and this change fixed it for me. https://stackoverflow.com/a/56989825/18080534

Reply:
Indeed, this seems to be the case.